apify-docs-theme/src/theme/DocBreadcrumbs/Items/Home/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import { translate } from '@docusaurus/Translate';
+import IconHome from '@theme/Icon/Home';
+import styles from './styles.module.css';
+
+export default function HomeBreadcrumbItem() {
+    const baseUrl = useBaseUrl('/');
+
+    const currentPath = window.location.pathname.replace(`^${baseUrl}`, '');
```

Comment:
you will need `useLocation()` here so it works on server side too

https://docusaurus.io/docs/advanced/routing#generating-and-accessing-routes